package.json

```diff
@@ -69,7 +69,17 @@
     "typography-plugin-code": "^0.16.20"
   },
   "overrides": {
+    "@enact/docs-utils": {
+      "documentation": {
+        "vue-template-compiler": "^1.0.28"
+      }
+    },
     "autolinker": "^4.0.0",
+    "axios": "^1.7.7",
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "react-server-dom-webpack@0.0.0-experimental-c8b778b7f-20220825": {
+      "react": "^18.3.1"
+    },
```

Comment:
i did not see any vulnerabilities related to these packages on local environment, after a deletion of node_modules and package-lock.json, on develop branch. Can you please double check?